apps/chat/src/components/Chat/Publish/HandlePublication.tsx

```diff
@@ -269,6 +273,10 @@ export function HandlePublication({ publication }: Props) {
     ? publication.targetFolder.replace(/^[^/]+/, 'Organization')
     : '';
 
+  const invalidEntities = deletedEntities.filter((entity) =>
+    publication.resources.map((r) => r.reviewUrl).includes(entity.id),
```

Comment:
I would suggest to change to
```suggestion
    publication.resources.some((r) => r.reviewUrl === entity.id),
```